app/src/main/java/com/it3048/mobiledeviceproject/MeetingAdapter.kt

```diff
@@ -7,21 +7,21 @@ import android.view.ViewGroup
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 
-class MeetingAdapter(val c:Context, val meetingList:ArrayList<MeetingDAO>):RecyclerView.Adapter<MeetingAdapter.meetingViewHolder>() {
-    inner class meetingViewHolder(val v:View):RecyclerView.ViewHolder(v){
+class MeetingAdapter(val c:Context, private val meetingList:ArrayList<MeetingDAO>):RecyclerView.Adapter<MeetingAdapter.MeetingViewHolder>() {
+    inner class MeetingViewHolder(val v:View):RecyclerView.ViewHolder(v){
```

Comment:
MeetingViewHolder should be capitalized, by the kotlin website standards: https://kotlinlang.org/docs/coding-conventions.html#choose-good-names
Also, meetingList should be private, so I made it so